hotsos/defs/scenarios/system/cve-needrestart.yaml

```diff
@@ -0,0 +1,35 @@
+checks:
+  has_needrestart_cve:
+    apt:
+      needrestart:
+        # Oracular
+        - min: '3.6-8'
+        - max: '3.6-8ubuntu4'
+        # Noble
+        - min: '3.6-7'
+        - max: '3.6-7ubuntu4.1'
+        # Jammy
+        - min: '3.5-5'
+        - max: '3.5-5ubuntu2.1'
+        # Focal
+        - min: '3.4-6'
+        - max: '3.4-6ubuntu0.1'
+        # Bionic
+        - min: '3.1-1'
+        - max: '3.1-1ubuntu0.1'
+        # Xenial
+        - min: '2.6-1'
+        - max: '2.6-1'
```

Comment:
I just wanted to make sure having all these releases here is actually useful before merging.

Reply:
doesnt hurt to have them.